### PR TITLE
Fix segment fault in arithmetic component.

### PR DESCRIPTION
**Pull request by Keepmoving-ZXY**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](https://github.com/primihub/community#development-workflow) to this repository.

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your master!
- [ ] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] Check your code additions will fail neither code linting checks nor unit test.
- [ ] Please provide a readable PR title.

### Description
1.Handle the segment fault when run arithmetic compoent;
2.Some code assume that column name only has one character, fix it.

### CODE REVIEW

**Comment by Keepmoving-ZXY** [comment]
I have read the CLA Document and I hereby sign the CLA

File: src/primihub/algorithm/arithmetic.cc [hunk 1]
```diff
@@ -33,10 +34,12 @@ ArithmeticExecutor<Dbit>::ArithmeticExecutor(
     : AlgorithmBase(dataset_service) {
   this->algorithm_name_ = "arithmetic";
 
-  auto& node_map = config.node_map;
-  // LOG(INFO) << node_map.size();
+  auto &node_map = config.node_map;
   std::map<uint16_t, rpc::Node> party_id_node_map;
   for (auto iter = node_map.begin(); iter != node_map.end(); iter++) {
+    if (iter->first == SCHEDULER_NODE)
```
Comment (phoenix20162016) [inline src/primihub/algorithm/arithmetic.cc:40]: 这个地方需要增加吗，我记得在生成partyconfig时同时过滤了

File: src/primihub/algorithm/arithmetic.cc [hunk 2]
```diff
@@ -94,23 +85,79 @@ ArithmeticExecutor<Dbit>::ArithmeticExecutor(
...
+    std::string party_name = party_info.first;
+    party_info_.insert(std::make_pair(party_info.first, party_id));
+
+    LOG(INFO) << "Party " << party_name << " has party id " << party_id << ".";
+  }
+}
+
 template <Decimal Dbit>
 int ArithmeticExecutor<Dbit>::loadParams(primihub::rpc::Task &task) {
+  LOG(INFO) << task.DebugString();
```
Comment (phoenix20162016) [inline src/primihub/algorithm/arithmetic.cc:117]: 这个日志级别改的高点吧

File: src/primihub/algorithm/arithmetic.cc (hunk 1 above)
Comment (Keepmoving-ZXY) [reply src/primihub/algorithm/arithmetic.cc]: 在构造PartyConfig的时候过滤了，我在代码中去掉这个检测了。

File: src/primihub/algorithm/arithmetic.cc (hunk 2 above)
Comment (Keepmoving-ZXY) [reply src/primihub/algorithm/arithmetic.cc]: 调试时候加的输出，忘记删除了，已经在代码中去掉了。

**Review by phoenix20162016 — APPROVED** [review]

**Pull request merged**